src/lazy/encoder/writer.rs

```diff
@@ -203,20 +202,16 @@ impl<'a, V: ValueWriter> ApplicationValueWriter<'a, V> {
 }
 
 impl<'a, 'value, 'top> ApplicationValueWriter<'a, BinaryValueWriter_1_1<'value, 'top>> {
-    pub fn config(&self) -> ValueWriterConfig {
-        self.raw_value_writer.config()
-    }
-
     pub fn with_container_encoding(mut self, container_encoding: ContainerEncoding) -> Self {
-        self.raw_value_writer = self
-            .raw_value_writer
```

Comment:
🗺️ These used to delegate to their inner raw writer, but I realized that was just storing application-level config at a lower level. Now the application-level types store their own config.